From: Sruthi Chandran <srud@disroot.org>
Date: Mon, 7 Jan 2019 11:52:29 +0530
Subject: Inhibit test for SSL

---
 spec/server_spec.rb | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/spec/server_spec.rb b/spec/server_spec.rb
index f173bfe..e163a6b 100644
--- a/spec/server_spec.rb
+++ b/spec/server_spec.rb
@@ -118,7 +118,7 @@ RSpec.describe Capybara::Server do
     Capybara.server = :default
   end
 
-  it 'should support SSL' do
+  xit 'should support SSL' do
     key = File.join(Dir.pwd, 'spec', 'fixtures', 'key.pem')
     cert = File.join(Dir.pwd, 'spec', 'fixtures', 'certificate.pem')
     Capybara.server = :puma, { Host: "ssl://#{Capybara.server_host}?key=#{key}&cert=#{cert}" }
